### PR TITLE
Map

### PR DESCRIPTION
**Pull request by anyahuang**
add map

### CODE REVIEW

File: map/urls.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+from django.urls import path
+from map import views
+
+appname = 'map'
```
Comment (SyncopatedPandemonium) [inline map/urls.py:5]: It should be app_name :)

Comment (SyncopatedPandemonium) [reply map/urls.py:5]: ❤️

File: map/templates/map/default.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{% extends 'post/base.html' %}
+
+{% block content %}
+
+<style>
```
Comment (SyncopatedPandemonium) [inline map/templates/map/default.html:5]: We should put style in style.css file, or create the new map.css and put it there.
We can do it later with refactoring though:)

**Review by SyncopatedPandemonium — APPROVED** [review]

**Review by MarieEngel — APPROVED** [review]
Like it :)

**Pull request merged**